SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -29,16 +29,15 @@ For the compiled portion of `spdlog`, the `SPDLOG_API` C++ macro indicates the s
 
 ### API Stability Policy [1.iv]
 
-There is no published policy for API stability in `spdlog`.
+There is no policy for API stability. However, the vendored package `spdlog_vendor` package importing the `spdlog dependency` is using a fixed version, in this case, the [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0), which will provide API stability to downstream packages.
```

Comment:
```suggestion
There is no policy for API stability. However, the vendored package `spdlog_vendor` package importing the `spdlog dependency` is using a fixed version, in this case [1.5.0](https://github.com/gabime/spdlog/releases/tag/v1.5.0), which will provide API stability to downstream packages.
```

Also, does it even make sense to mention the specific version here. Perhaps instead you should link to where the current version can be found?